### DISCUSSION TITLE
Show the Brightness percentage

### DISCUSSION THREAD

**Discussion by Rqbln** [polybar/polybar#2955] [Q&A] [answered]
Hello, i tried to replicate the volume code from the original config file and use it for showing brightness, but it doesn't work, is someone know how to do it ?

Volume original code :
```
[module/pulseaudio]
type = internal/pulseaudio

format-volume-prefix = "VOL "
format-volume-prefix-foreground = ${colors.primary}
format-volume = <label-volume>

label-volume = %percentage%%

label-muted = muted
label-muted-foreground = ${colors.disabled}
```

My code for brightness :
```
[module/backlight]
type = internal/backlight
format-brightness-prefix = "LUM "
format-brightness-prefix-foreground = ${colors.primary}
format = <label-brightness>

label-brightness = %percentage%%
```

**Comment by patrick96** [comment]
Those are not the setting names in the backlight module. Just use `format-prefix` (not `format-brightness`) and the label is called `label` (not `label-brightness`).

https://github.com/polybar/polybar/wiki/Module:-backlight

**Reply by Rqbln** [reply]
I tried with the wiki with this code, but still doesn't work :

```
[module/backlight]
type = internal/backlight

format-prefix = "LUM "
format-prefix-foreground = ${colors.primary}

card = intel_backlight
use-actual-brightness = true
enable-scroll = true

label = %percentage%
label-foreground = ${colors.foreground}
```

Brightness is just not showing on polybar

**Reply by patrick96** [reply]
Does polybar print any warnings or errors to the terminal? Did you add the module to one of the three module lists in the bar section?

**Reply by Rqbln** [reply]
I have no errors or warnings and i tried right, left and center

**Reply by patrick96** [reply]
Can you post the output polybar does produce?

**Reply by Rqbln** [reply]
Ok, i found the errors x)
I didn't found them because i was launching polybar at startup
here is the output : 
```
notice: Loading module 'backlight' of type 'internal/backlight'
error: Disabling module "backlight" (reason: The file '/sys/class/backlight/intel_backlight/actual_brightness' does not exist)
```
I think manjaro kde doesn't use this file to manage brightness

**Reply by patrick96** [reply]
Look in the folder `/sys/class/backlight/`. It should show you the names of the brightness controllers in your system.

If it is empty, you probably can't change the brightness. That's to be expected on a desktop.

**Reply by Rqbln** [reply, marked as answer]
it is `/sys/class/backlight/amdgpu_bl0/actual_brightness ` and not `/sys/class/backlight/intel_backlight/actual_brightness`
How can i change this in polybar settings ?

Nvm, juste changed the `card` setting from `card = intel_backlight` to `card = amdgpu_bl0`

Thanks a lot :)
